src/utilities/theme/types.ts

```diff
@@ -36,6 +36,8 @@ export interface RoleColors {
 
 export type Role = keyof RoleColors;
 
+export type Mode = 'light' | 'dark';
```

Comment:
What do you think about an `'inverse'` type? It’s not really a mode and wouldn’t be a valid type for the root theme provider, but as a consumer this would be easier to use than getting the mode from context and using the inverse function or writing a ternary. If not, the theme hook/context consumer needs to be exported and documented.

Reply:
Definitely 👍